Sources/Commands/SwiftTool.swift

```diff
@@ -249,7 +234,7 @@ public class SwiftTool {
     /// Helper function to get package root or throw error if it is not found.
     func getPackageRoot() throws -> AbsolutePath {
         guard let packageRoot = packageRoot else {
-            throw Error.rootManifestFileNotFound
+            throw StringError("Could not find \(Manifest.filename) in this directory or any of its parent directories.")
```

Comment:
With the new diagnostics, is the best-practice pattern in general to include the message immediately, or to create and throw a type that captures just the parameters and can then general the message on the fly?